daemon/logger/splunk/splunk.go

```diff
@@ -0,0 +1,256 @@
+// +build linux
```

Comment:
Supporting Windows is not supported right now as the WIndows support is till being developed. Once it settles, we'll revisit.